packages/adapter-node/README.md

```diff
@@ -94,23 +94,17 @@ MY_ORIGINURL=https://my.site \
 node build
 ```
 
-### xForwardedForIndex
+### `XFF_DEPTH`
 
-If the `ADDRESS_HEADER` is `X-Forwarded-For`, the header value will contain a comma-separated list of IP addresses. For example, if there are three proxies between your server and the client, proxy 3 will forward the addresses of the client and the first two proxies:
+If the `ADDRESS_HEADER` is `X-Forwarded-For`, the header value will contain a comma-separated list of IP addresses. The `XFF_DEPTH` environment variable should specify how many trusted proxies sit between your server and the client. E.g. if there are three trusted proxies, proxy 3 will forward the addresses of the client and the first two proxies:
```

Comment:
What if we said this, so that we don't need to worry about what proxies sit in front of the client? Or is that too confusing given the possibility of apps running on intranets etc?

```suggestion
If the `ADDRESS_HEADER` is `X-Forwarded-For`, the header value will contain a comma-separated list of IP addresses. The `XFF_DEPTH` environment variable should specify how many trusted proxies sit between your server and the public internet. E.g. if there are three trusted proxies, proxy 3 will forward the addresses of the client and the first two proxies:
```

Reply:
yeah, much better. i like the first one

Reply:
I'm not sure I understand the difference between these and what I said, but @mrkishi's wording is fine with me if you'd like to change it to that